content-src/components/DiscoveryStreamComponents/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export class Topic extends React.PureComponent {
+  render() {
+    const {url, name} = this.props;
+    return (<li><a key={name} href={url}>{name}</a></li>);
+  }
+}
+
+export class Navigation extends React.PureComponent {
+  render() {
+    const {links} = this.props || [];
+    const {alignment} = this.props || "centered";
+    const moreRecs = links.pop();
+    return (
+      <span className={`ds-navigation ds-navigation-${alignment}`}>
```

Comment:
Any concerns here about concatenating server provided string?

Reply:
Not that I'm aware of. I don't *think* a server defined variable can escape the back quote.

All it can do is define some pretty funky css classnames, right?

For what it's worth we're doing this in other places too, so if it is an issue we should probably flag it for multiple spots in other components too.